website/docs/components/select/select-a11y.md

```diff
@@ -51,13 +51,13 @@ The following list will help you to keep in mind the necessary roles and attribu
 
 Table: Roles and attributes
 
-| Component            | Attribute                       | Usage                                                                                         |
-| -------------------- | ------------------------------- | --------------------------------------------------------------------------------------------- |
-| `Select.InputSearch` | `aria-describedby="IDREF"`      | Define an accessible description announcing the number of search results. Refer to the [options filtering example](./select-code.md#options-filtering). |
+| Component            | Attribute                  | Usage                                                                                                                                                    |
+| -------------------- | -------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `Select.InputSearch` | `aria-describedby="IDREF"` | Defines an accessible description announcing the number of search results. Refer to the [options filtering example](./select-code.md#options-filtering). |
```

Comment:
@j-mnizhek it was an intentional imperative, because they have to do that themselves

Reply:
It’s a little bit confusing that the documentation pattern changes. In the table above, we describe what a specific role or attribute does, while in this table, we provide a recommendation.

Reply:
If we change it from "define" to "defines", then the whole sentence should be rewritten, because obviously aria-describedby per se doesn't have anything to do with search results. I wanted to write a specific recommendation on how to use the property in this specific case. 

Then it should be something like "Defines an accessible description for the element. Use it to announce the number of search results." What do you think about this version?

Reply:
Got it! Yeah! I think this version will be better 👍

Reply:
Got it! Yeah! I think this would suit better 👍
(it's strange I've sent this comment 2 hours ago and now I see it disappeared somehow)

Reply:
this PR is just too big and cursed, it happened to me as well